apps/plugin-browser/src/pages/popup/popup-contents/action-center/text-field-with-dark-mode.tsx

```diff
@@ -1,24 +1,33 @@
 import { TextField, TextFieldProps } from "@hashintel/design-system";
 import { outlinedInputClasses } from "@mui/material";
 
+import {
+  darkModeBorderColor,
+  darkModeInputBackgroundColor,
+  darkModeInputColor,
+  darkModePlaceholderColor,
+} from "../../../shared/dark-mode-values";
+
 export const TextFieldWithDarkMode = (props: TextFieldProps) => {
   return (
     <TextField
       {...props}
       InputProps={{
-        sx: ({ palette }) => ({
+        sx: () => ({
           "@media (prefers-color-scheme: dark)": {
```

Comment:
Maybe not worth addressing, but we could set `palette.mode` to `"dark"` in the theme we pass to the `ThemeProvider`, so we can check the value of this boolean rather than using a media query every time we want to make styling dependent on dark mode. Here's [an example](https://mui.com/material-ui/customization/dark-mode/#system-preference) from their docs.

Reply:
Thank you – I've created H-1029 to do this in a follow up.